calc_photodisintegration.py

```diff
@@ -121,18 +106,36 @@
         fmt='%i\t%i\t%06d' + '\t%.9g' * 201,
         header=header)
 
+if __name__ == "__main__":
+    fields = [
+        photonField.CMB(),
+        photonField.EBL_Kneiske04(),
+        photonField.EBL_Stecker05(),
+        photonField.EBL_Franceschini08(),
+        photonField.EBL_Finke10(),
+        photonField.EBL_Dominguez11(),
+        photonField.EBL_Gilmore12(),
+        photonField.EBL_Stecker16('upper'),
+        photonField.EBL_Stecker16('lower'),
+        photonField.URB_Protheroe96(),
+        photonField.URB_Fixsen11(),
+        photonField.URB_Nitu21()
+        ]
+    
+    for field in fields:
+        processRate(field)
+
+
 
 # ----------------------------------------------------
 # Calculate photon emission probabilities
 # ----------------------------------------------------
-fields = [
-    photonField.CMB(),
-    photonField.EBL_Gilmore12(),
-    photonField.URB_Protheroe96()
-    ]
+def processEmission(field):
+    """ calculate photon emission probabilities """
 
-for field in fields:
-    print(field.name)
+    folder = "data/Photodisintegration/"
+    if not os.path.isdir:
```

Comment:
Why can't we use the test as in the oher files?
`if not os.path.exists(folder):`


Maybe I just don't get what you want to check here...

Reply:
In this context this would be the same test. The function `isdir` also checkes wether it is a file or a directory. But in this case it is already defined by the folder naming, that it should be a folder.